.github/workflows/sync-triage-config.yml

```diff
@@ -4,6 +4,11 @@ on:
   push:
     branches:
       - master
+
+concurrency:
+  group: sync-triage-config
+  cancel-in-progress: true
```

Comment:
@MikeMcQuaid since this is a workflow-level `concurrency`, it won't stop the job from running on each repo at the same time, right?

Reply:
@Rylan12 Yes, I believe so.